### ISSUE TITLE
Automate checking for updates?

### ISSUE THREAD

**Issue by simonrob** [blacktirion/email-oauth2-proxy-docker#46]
Thanks for creating this integration – I really appreciate someone else taking on maintaining Docker support, as it's not something I use with the proxy personally.

I do still occasionally get queries/issues on the main proxy repository about this. The [most recent one](https://github.com/simonrob/email-oauth2-proxy/pull/100#issuecomment-1926464169) was because the Docker image wasn't quite up to date. I quite like the current approach you have of opening an issue to trigger a new build, but wondered if there might be a way to further automate this.

Looking into it a little further, I found [this example action](https://github.com/orgs/community/discussions/26323#discussioncomment-8345691), where the docker repository just checks the main one regularly for updates. Would this work here?

**Comment by moriahmorgan** [comment]
Hey @simonrob, thanks for your request. I haven't been ignoring it, but I started a new role at another company so I've been focused on that. Let me see what I can do, for sure. I will leave this open.

**Comment by simonrob** [comment]
Just a thought – I released a new version of the proxy today, and realised that as part of this I could simply add a GitHub Action on my side that just opens a new issue on your repository to trigger an update. I'll look into this if I get chance – seems like it may be [relatively straightforward.](https://github.com/marketplace/actions/open-issue-action).

**Comment by shymega** [comment]
Agreed - it would be good to link this in the CI Action, so that we don't have to have issues opened for updates. - this is noisy.

An approach like this CI workflow I implemented for Android Boot Manger could be an option: https://github.com/Android-Boot-Manager/droidboot_platforms/blob/main/.github/workflows/submodule_dispatch_workflow.yml

Using GitHub's fine-grained tokens, this would also be relatively secure. It would require @simonrob to be added to this repo, and then to add a fine-grained token with the Actions+Workflow read/write scopes to their account, and added as a secret to the workflow, which would run in the `email-oauth2-proxy` repository upon the `tag:` event.

**Comment by simonrob** [comment]
Thanks for following up - sorry to hear that, and I hope you're doing better now.

In all honesty, this has become less important because there just isn't much need to update the proxy these days - it's pretty much complete/finished in many ways. So I think it's probably only worth doing if it's really straightforward to set up.

Let me know what you think?
